webapp/channels/src/actions/telemetry_actions.jsx

```diff
@@ -204,25 +204,39 @@ export function trackPluginInitialization(plugins) {
     });
 }
 
+function getTopThreeSelectors(selectors) {
+    const topThreeSelectors = [];
+    for (const selector of selectors) {
+        if (topThreeSelectors.length === 3) {
```

Comment:
Is the optimization of only returning the first 3 values here worth the extra complexity over just writing this as `selectors.filter((selector) => selector.calls > 5)`?

Reply:
I'll make this change

Reply:
done